plugins/woocommerce-admin/client/task-lists/fills/PaymentGatewaySuggestions/test/index.js

```diff
@@ -72,6 +72,16 @@ const paymentGatewaySuggestions = [
 		is_visible: true,
 		recommendation_priority: 1,
 	},
+	{
+		id: 'woocommerce_payments:bnpl',
+		title: 'Activate BNPL instantly on WooPayments',
```

Comment:
More of a product/design question, but is BNPL a widely known term for merchants? I'm just wondering if it could be confusing.

Reply:
I guess it is a known acronym. I trust that Design and Marketing know the "language" of merchants.